components/automate-ui/src/app/entities/data-bags/data-bag-details.effects.ts

```diff
@@ -43,4 +46,31 @@ export class DataBagItemsEffects {
       });
     }));
 
+  @Effect()
+  updateDataBagItem$ = this.actions$.pipe(
+    ofType(DataBagItemsActionTypes.UPDATE),
+    mergeMap(({ payload: { dataBagItem } }: UpdateDataBagItem) =>
+      this.requests.updateDataBagItem(dataBagItem).pipe(
+        map((resp: DataBagItems) => new UpdateDataBagItemSuccess(resp)),
+        catchError((error: HttpErrorResponse) =>
+          observableOf(new UpdateDataBagItemFailure(error))))));
+
+  @Effect()
+  updateDataBagItemSuccess$ = this.actions$.pipe(
+    ofType(DataBagItemsActionTypes.UPDATE_SUCCESS),
+    map(({ payload: dataBagItem }: UpdateDataBagItemSuccess) => new CreateNotification({
+    type: Type.info,
+    message: `Successfully Updated Data Bag Item - ${dataBagItem.name}.`
```

Comment:
Let's be consistent with the majority of the code base
```suggestion
    message: `Updated data bag item ${dataBagItem.name}.`
```

Reply:
@msorens I have incorporated review comment changes, please have a look.